factory/webServerFactory.go

```diff
@@ -9,6 +10,10 @@ import (
 
 // CreateWebServer will create a new instance of core.WebServerHandler
 func CreateWebServer(apiConfig config.ApiRoutesConfig, statusMetricsHandler core.StatusMetricsHandler) (core.WebServerHandler, error) {
+	if check.IfNil(statusMetricsHandler) {
```

Comment:
not really needed here, as it is only forwarded to NewMetricsFacade

Reply:
it cam be removed.